data/transition-sites/phe_empho.yml

```diff
@@ -1,8 +1,9 @@
 ---
 site: phe_empho
 whitehall_slug: public-health-england
-homepage: https://www.gov.uk/government/organisations/public-health-england
+homepage: http://www.empho.org.uk
```

Comment:
The `homepage` shouldn't be the same as the `host`, because if we switched this site back to using mappings in future we'd cause a redirect loop. For other global sites we've left the `homepage` as the org homepage or the same URL as the global redirect.